Lab/src/main/java/agh/ics/oop/AbstractWorldMap.java

```diff
@@ -18,7 +18,8 @@ public boolean place(Animal animal) {
             this.animals.put(animal.getPosition(),animal);
             return true;
         }
-        return false;
+        else{
+        throw new IllegalArgumentException("Cant place animal on "+animal.getPosition());}
```

Comment:
skoro rzucamy wyjątkiem to nie zwrócimy tu nigdy false -> ta metoda nie powinna już zwracać boolean